docs/_templates/quickstart.rst

```diff
@@ -0,0 +1,69 @@
+.. _quick_start:
+
+Quick Start
+===========
+
+This quick-start contains an overview of how the ``xpdAn`` software works.
+To understand more details about it, please refer to the detailed documentation in :ref:`xpdu`
+
+Please use this page as a reminder of the workflow and to copy & paste code snippets into the
+ipython terminals that are controlling your experiment and analysis.  After
+pasting, hit enter.
+
+Remember, to post questions about anything XPD, including software, and to see archived answers, at the `XPD-Users Google group
+<https://groups.google.com/forum/#!forum/xpd-users;context-place=overview>`_ . If you are not already a member please request to join
+the community
+
+XPD Data Analysis Workflow Summary
+------------------------------------
+
+This is the summary of the steps for analysis data at XPD. They are explained below.
+Carry out the steps in this order to ensure a successful analysis.
+
+  1. If you haven't already, join `XPD-Users Google group <https://groups.google.com/forum/#!forum/xpd-users;context-place=overview>`_ . Look here for answers if you get stuck and if the answer to your question is not already there, please ask it!
+
+  2. If not started already start a proxy and an analysis session.
+  To start the proxy run ``bluesky-0MQ-proxy {{ bs_proxy_config }}``.
+  To start the analysis session run ``{{ analysis_session_cmd }}``.
+
+  3. Start the analysis by running ``{{ start_analysis }}``
+
+  5. Your data will be automatically saved and visualized via the analysis pipleine.
+
+    * The data will be saved in ``.../xpdUser/tiff_base/<Sample_name_from_spreadsheet>``.
+
+    * The pipeline will save:
+
+        1. Dark corrected image (as ``.tiff``)
+        2. Mask (as ``.msk`` and ``.npy``)
```

Comment:
is one mask saved per image?  If not, mention.  Also, are two mask files saved?  this is kind of implied.  What is the differences?

Reply:
For every mask we produce we save it in the two different formats.
How often the masks are produced is controlled by the mask settings.

Reply:
@CJ-Wright just to be clear, my questions were not for my personal edification but questions that a user might have, so should be answered in the docs as it were.    This is true for all the questions I posted.....I just want to make sure that  there was no confusion on that....

Reply:
ok, no problem.